frontend/src/lib/lemon-ui/LemonSegmentedButton/index.ts

```diff
@@ -0,0 +1 @@
+export * from './LemonSegmentedButton'
```

Comment:
I was of the impression, we don't add `index.ts` files and rather import from the component file directly. Related - should we add an export to '@posthog/lemon-ui'?

Reply:
We definitely don't do `index.ts` for the components themselves, since that was a nightmare when searching for files, but IMO the experience of importing from `.../Component` is nicer than `.../Component/Component`. What do you think?
Thanks for reminding about `@posthog/lemon-ui`, added.